src/datasources/email-template/pushwoosh-template.service.ts

```diff
@@ -1,24 +1,71 @@
 import { Injectable } from '@nestjs/common';
 import { getAddress } from 'viem';
 import { IEmailTemplate } from '@/domain/interfaces/email-template.interface';
+import { Chain } from '@/routes/chains/entities/chain.entity';
+import { readFileSync } from 'fs';
+import { join } from 'path';
 
 @Injectable()
 export class PushWooshTemplate implements IEmailTemplate {
-  addressToHtml(address: string): string {
+  addressToSafeWebAppUrl(args: { chain: Chain; safeAddress: string }): string {
+    return `https://app.safe.global/home?safe=${args.chain.shortName}:${args.safeAddress}`;
```

Comment:
Does it make sense to set this base url via configuration? I mean, could it be a scenario where the Safe Web App lives on a different base url? (staging/development environments for example)

Reply:
I've added a `safeWebApp.baseUri` configuration value in a40eb1ca59a440ffd68a3a4a1316dc59cc05fe0f.